url.bs

```diff
@@ -3172,23 +3180,26 @@ work.
         "publisher": "Unicode Consortium"
     },
     "UTS36": {
-      "href": "http://unicode.org/reports/tr36/",
-      "authors" : ["Mark Davis", "Michel Suignard"],
-      "title": "Unicode Security Considerations",
-      "publisher" : "Unicode Consortium"
+        "href": "http://www.unicode.org/reports/tr36/",
+        "authors": ["Mark Davis", "Michel Suignard"],
+        "title": "Unicode Security Considerations",
+        "publisher": "Unicode Consortium"
     }
 }
 </pre>
 
 <pre class=anchors>
-urlPrefix: https://w3c.github.io/FileAPI/; type: dfn
-    text: blob url store; url: #BlobURLStore
-urlPrefix: https://w3c.github.io/media-source/#idl-def-; type: interface
-    text: MediaSource
-urlPrefix: https://w3c.github.io/mediacapture-main/#idl-def-; type: interface
-    text: MediaStream
-url: http://www.unicode.org/reports/tr46/#ToASCII; type: dfn; text: toascii; spec: IDNA
-url: http://www.unicode.org/reports/tr46/#ToUnicode; type: dfn; text: tounicode; spec: IDNA
+# XXX Cannot use spec attribute here, as a Bikeshed bug currently prevents it
+# from coalescing autolinked and manually linked FILEAPI entries in the index.
```

Comment:
Ah, found the issue - FileAPI is, per usual in W3C, a versioned spec, so its full shortname is `fileapi-1`. However, the biblio name is spelled "FileAPI". They happened to run down slightly different codepaths that stored them separately, but then both get corrected to the biblio spelling upon printing.

I've now fixed it to coalesce properly.